docs/installation.md

```diff
@@ -55,9 +55,11 @@ Under **Arch** Linux :
 
 ```sh
 pacman -S nodejs npm qpdf imagemagick pdfminer tesseract python-pip
-pip install camelot-py[cv] pdfminer.six numpy pillow scikit-image PyPDF2
```

Comment:
why removing pdfminer.six ?

Reply:
The line above: 
`pacman -S nodejs npm qpdf imagemagick pdfminer tesseract python-pip`
already installs pdfminer.
The above package is based on pdfminer.six [link](https://www.archlinux.org/packages/community/any/python-pdfminer/).
However, there's a typo: let me fix.

Reply:
> However, there's a typo: let me fix.
Done in fead78d

Reply:
nice! I didn't realize it was on pacman